docs/HDRUK/3.0.0.form.json

```diff
@@ -1707,154 +1515,6 @@
                   },
                   "location": "structuralMetadata.syntheticDataWebLink"
             },
-            {
-                  "required": true,
-                  "title": "Age grouping",
-                  "description": null,
-                  "guidance": "",
-                  "examples": [
-                        "30-34 years"
-                  ],
-                  "is_list": false,
-                  "is_optional": false,
-                  "types": {
-                        "type": "string",
-                        "options": [
```

Comment:
These entries are important because the `options` are used to populate drop down menus on the front end form.  They don't look related to the changes you've made so is might be that an earlier change to the `create_markdown.py` script has caused these fields to disappear when the `3.0.0.form.json` file is regenerated.